spec/spec_helper.rb

```diff
@@ -19,6 +19,9 @@
 require 'coveralls'
 Coveralls.wear!('rails')
 
+require_relative 'support/user_sign_in_helper.rb'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.